pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1960,6 +1960,31 @@
     ::
       closed-connections
     ==
+  ::
+  ?:  ?=(%code-changed -.task)
+    ~>  %slog.[0 leaf+"eyre: code-changed: throwing away old cookies"]
+    =.  authentication-state.server-state.ax  *authentication-state
+    ::
+    =/  event-args  [[our eny duct now scry-gate] server-state.ax]
+    =*  by-channel  by-channel:(per-server-event event-args)
+    =*  channel-state  channel-state.server-state.ax
+    ::
+    =/  channel-ids=(list @t)  ~(tap in ~(key by session.channel-state))
+    ?:  =(~ channel-ids)
+      [~ http-server-gate]
```

Comment:
I don't think it's very important to catch the "do nothing" case early, the overhead from the couple lines below is extremely minimal. Might be cleaner to remove this, so you only have a single exit point in this block of code.